pgmq-extension/test/sql/base.sql

```diff
@@ -284,10 +284,9 @@ SELECT pgmq.create('detach_archive_queue');
 DROP EXTENSION pgmq CASCADE;
 SELECT tablename FROM pg_tables WHERE schemaname = 'pgmq' AND tablename = 'a_detach_archive_queue';
 
--- With detach, archive remains
+-- queues and archive remains
 CREATE EXTENSION pgmq;
 SELECT pgmq.create('detach_archive_queue');
-SELECT pgmq.detach_archive('detach_archive_queue');
 DROP EXTENSION pgmq CASCADE;
 SELECT tablename FROM pg_tables WHERE schemaname = 'pgmq' AND tablename = 'a_detach_archive_queue';
 
```

Comment:
```suggestion
-- queues and archive remains
CREATE EXTENSION pgmq;
SELECT pgmq.create('queue_remains');
DROP EXTENSION pgmq CASCADE;
SELECT tablename FROM pg_tables WHERE schemaname = 'pgmq' AND tablename = 'q_queue_remains';
SELECT tablename FROM pg_tables WHERE schemaname = 'pgmq' AND tablename = 'a_queue_remains';
```

Reply:
made this change and to the expected output in https://github.com/pgmq/pgmq/pull/413/commits/70b8a4f746d0a3d6ea7a7a1ec857f29c387613a0